inventory/dynamic/gcp/hosts.py

```diff
@@ -0,0 +1,402 @@
+#!/usr/bin/env python
```

Comment:
any reason to rename this file?  This looks like gce.py.  I'd prefer if we kept the same name.

Reply:
I want to make it clear what it's for, and the naming is redundant. It has additional patches on top of gce.py.  I can rename it back, but I could also add a comment at the top that talks about its provenance.

Reply:
Ok.  This should be fine.  Adding the comment and what the expectation is around keeping this file updated when a newer gce.py comes out.